packages/edit-post/src/components/header/style.scss

```diff
@@ -4,6 +4,7 @@
 	background: $white;
 	display: flex;
 	flex-wrap: wrap;
+	flex-direction: row-reverse;
```

Comment:
Interesting approach, I wouldn’t think that you can tweak DOM order and change it back visually CSS to make the experience better for keybord-only users 😃

I’m looking forward to the next plugin release to see how it plays out.

Reply:
Yes, I'm curious. It's debatable whether this is ok visually, but the large gap between the two areas and the fact that it's almost split at the column boundary may make it ok.